examples/force_control/Force_control_example.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Inverse kinematics with the Kuka robot where the goal is to follow a moving sphere.
+"""
+
+import numpy as np
+from itertools import count
+
+from pyrobolearn.simulators import Bullet
+from pyrobolearn.worlds import BasicWorld
+from pyrobolearn.robots import KukaIIWA, Body, sensors
+from pyrobolearn.utils.transformation import *
+
+from simulate_test_ur.plotting_ee_FT import EeFtRealTimePlot
```

Comment:
You did not include this python file and therefore we are not able to test the pull request functionality. May you please upload it? 
However, I removed the dependencies on this plotting functionality, and the code runs nicely! So, if you just do the suggested changes, we may merge the pull request :)